app/data/test_dates.json

```diff
@@ -0,0 +1,124 @@
+{
```

Comment:
This adds a `test_dates` schema but we already have `test_date`—should these be combined?